### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RunicAce**
This update adds the FAQ from the discord.

### CODE REVIEW

**Review by Doomsdayrs — COMMENTED** [review]
Instead of using the Q / A format that works on discord, Use a more appropriate markdown way.

```
# Frequently Asked Questions

## How can I request a new source
Make sure there isn't another for the same one at the extensions repo, Otherwise https://github.com/shosetsuorg/extensions/issues/new/choose (SOURCE REQUEST)
```

File: src/help/faq/README.md [hunk 1]
```diff
@@ -5,5 +5,70 @@ lang: en-US
...
+Q: I have no Extensions, what do I do?
+A: please make sure you have the following options enabled.
+
+Settings > Download
+Settings > Updates
+- Download/Update on Metered Connection
+- Download/Update on low battery
+- Download/Update on low storage
+
+Links:
```
Comment (Doomsdayrs) [inline src/help/faq/README.md:64]: Why is this here?

Comment (RunicAce) [reply src/help/faq/README.md:64]: I just copy pasted. I'll reformat it, but ultimately it's up to you what you want to include. You could easily pull it afterwards, then delete what you deem unnecessary.
